realtek: cleanup LAG logging

plappermaul opened the pull request:
Setting up DSA bond silently fails if mode is not 802.3ad. Add log message
to fix it. As we are already here hamonize all logging messages in the
add/delete functions.

svanheule commented on target/linux/realtek/files-5.10/drivers/net/dsa/rtl83xx/common.c:
```diff
@@ -424,24 +424,30 @@ static int __init rtl83xx_get_l2aging(struct rtl838x_switch_priv *priv)
 	u32 algomsk = 0;
 	u32 algoidx = 0;
+
+	pr_info("%s: Adding port %d to LA group %d\n", __func__, port, group);
```
There's a pr_debug at the end of the function. These drivers are verbose enough as they are, I don't think this line really adds value, being so verbose all the time.

svanheule commented on target/linux/realtek/files-5.10/drivers/net/dsa/rtl83xx/common.c:
```diff
@@ -424,24 +424,30 @@ static int __init rtl83xx_get_l2aging(struct rtl838x_switch_priv *priv)
-/* Caller must hold priv->reg_mutex */
+/*
+ * Caller must hold priv->reg_mutex
+ */
```
This is a short line, doesn't need to be changed to a multi-line comment.

svanheule commented on target/linux/realtek/files-5.10/drivers/net/dsa/rtl83xx/common.c:
```diff
@@ -479,37 +485,39 @@ int rtl83xx_lag_add(struct dsa_switch *ds, int group, int port, struct netdev_la
-/* Caller must hold priv->reg_mutex */
+/*
+ * Caller must hold priv->reg_mutex
+ */
```
Same as above. Not really a useful change IMHO.

svanheule commented:
Thanks for improving the driver's feedback a bit.

checkpatch.pl also has some remarks about this patch:

```WARNING: line length of 108 exceeds 100 columns
#66: FILE: target/linux/realtek/files-5.10/drivers/net/dsa/rtl83xx/common.c:488:
+	pr_debug("%s: LA group %d members now %016llx.\n", __func__, group, priv->lags_port_members[group]);

WARNING: line length of 108 exceeds 100 columns
#105: FILE: target/linux/realtek/files-5.10/drivers/net/dsa/rtl83xx/common.c:520:
+	pr_debug("%s: LA group %d members now %016llx.\n", __func__, group, priv->lags_port_members[group]);

ERROR: Missing Signed-off-by: line(s)

total: 1 errors, 2 warnings, 89 lines checked
```

svanheule reviewed (commented):
Any reason you went with "LA group" instead of just "LAG"? The former does no appear to be used elsewhere in the kernel.

plappermaul commented:
> Any reason you went with "LA group" instead of just "LAG"? The former does no appear to be used elsewhere in the kernel.

This comes from the old pr_info() at the beginning of the functions.

plappermaul committed:
realtek: cleanup LAG logging

Setting up DSA bond silently fails if mode is not 802.3ad. Add log message
to fix it. As we are already here harmonize all logging messages in the
add/delete functions.

Signed-off-by: Markus Stockhausen <markus.stockhausen@gmx.de>

plappermaul commented:
changed everything to LAG

svanheule commented:
Thanks! Merged into master with commit 78b7be9f6c6961b3c71f213c9e4bde18af5c8ee6.
